packages/codemirror-graphql/resources/watch.js

```diff
@@ -101,14 +101,14 @@ function checkFiles(filepaths) {
     .then(testSuccess =>
       lintFiles(filepaths).then(lintSuccess =>
         typecheckStatus().then(
-          typecheckSuccess => testSuccess && lintSuccess && typecheckSuccess
-        )
-      )
+          typecheckSuccess => testSuccess && lintSuccess && typecheckSuccess,
+        ),
+      ),
```

Comment:
I've had my head out the game for a while. Why add trailing commas?

Reply:
it’s on trend now because es2018 i think adoped trailing function param commas

Reply:
makes git diffs cleaner, leas merge conflicts. es5 the setting before did it for just literals, all is literally everything else haha